gatekeeper/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,29 @@
+// Package bootstrap provides authentication actions for the bootstrap process
+package bootstrap
+
+import (
+	"fmt"
+
+	"github.com/manifoldco/torus-cli/gatekeeper/apitypes"
+	"github.com/manifoldco/torus-cli/gatekeeper/bootstrap/aws"
+)
+
+const (
+	// AWSPublic is Amazon's Public Cloud Provider
+	AWSPublic = "aws"
+)
+
+// Handler the function returned by the Bootstrap server that can
```

Comment:
I don't know how I feel about this type, and returning it from `bootstrap.New` directly. Thoughts?